### PR TITLE
Add blog about dropping java 8

### PR DESCRIPTION
**Pull request by davsclaus**
Review and feedback welcome about that we drop java 8.
I wanted to keep it short and about java 8 only.

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: content/blog/2021/09/eol-java8/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Java 8 to be dropped by end of 2021"
+date: 2021-09-21
+authors: [davsclaus]
+categories: ["Releases"]
+preview: All the Camel projects will drop support for Java 8 by end of 2021
```
Comment (gzurowski) [inline content/blog/2021/09/eol-java8/index.md:6]:
```suggestion
preview: All Camel projects will drop support for Java 8 by end of 2021
```

File: content/blog/2021/09/eol-java8/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+All the Camel projects will drop support for Java 8 by end of 2021.
```
Comment (gzurowski) [inline content/blog/2021/09/eol-java8/index.md:9]:
```suggestion
All Camel projects will drop support for Java 8 by end of 2021.
```

File: content/blog/2021/09/eol-java8/index.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+The next LTS release Camel 3.14, scheduled for December 2021, will be the last
+release to support Java 8.
+
+In light of this we will extend the support period of Camel 3.14 from 1-year to 2-years.
```
Comment (gzurowski) [inline content/blog/2021/09/eol-java8/index.md:14]:
```suggestion
In light of this we will extend the support period of Camel 3.14 from 1 year to 2 years.
```

**Review by gzurowski — APPROVED** [review]
LGTM 👍

**Review by aldettinger — APPROVED** [review]

**Pull request merged**